### DISCUSSION TITLE
Setting up FDO

### DISCUSSION THREAD

**Discussion by Antibas** [secure-device-onboard/pri-fidoiot#698] [General]
Hello,

I am trying to set **FDO** up on my computer with a client-server setup. However I find the documentation not to be beginner-friendly and, as a beginner myself, I struggle to understand the `README.md` files. Even after reading the specification and watched videos uploaded on Youtube, I still have a problem to understand how it is set up. 

Is there something I missed? Is there a `README.md` or video that explains it clearly and I missed it?

Thank you in advance!

**Comment by ben-krieger** [comment]
@Antibas I'm not sure what your goals are, but if you just want to kick the tires on FDO and see the protocol messages passed, I just released a Go library in this organization. It has an example client and server, which you can copy and modify as you like.

Hopefully you won't have the same setup issues as with the Java version, because it doesn't require a separate running database and the configuration is simple command line flags. It can be built with nothing but the Go 1.23 toolchain (unless you use the TPM simulator - then you need openssl and gcc, too).

Let me know if it meets your needs and how easy it is to try out. (It was just released today and documentation will be improved over time.)

**Comment by Sai-Anudeep47** [comment]
Hi @Antibas . You can refer this [README](https://github.com/fido-device-onboard/pri-fidoiot/blob/master/README.md) to get started with FDO. This includes prerequisites, system requirements, commands to start a server, client and complete an onboarding. Please feel free to include logs / screenshots if you face any issues.
